sdks/python/apache_beam/transforms/trigger_test.py

```diff
@@ -81,16 +84,8 @@ def bundle_data(data, size):
 
     if not groupings:
       groupings = [1]
+    grouping = list(groupings) + [-group_by for group_by in groupings]
```

Comment:
<!--new_thread; commit:23bbd5d9bf76313060d10b6a040be8c452229d64; resolved:0-->
What is this?  This is currently a no-op.

Reply:
It expands grouping by also adding negative values, which also tests adding elements in reverse order.

Reply:
What I mean is that the variable `grouping` (without "s" at the end) is never used in the rest of the function.  Is this a typo?

Reply:
What I mean is that the variable `grouping` (without "s" at the end) is never used in the rest of the function.  Is this a typo?

Reply:
Oh, yes, nice eye!

And fixing this makes some of the non-deterministic ones fail... Enabling only explicitly on some tests.